added coiote tutorials (#164)
AVSystem/developer-zone: docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md

> # What collected IoT data can you find in the Coiote?

Reviewer: Sound weird...
```suggestion
# What IoT data is collected and stored in the Coiote?
```
or `What types of IoT data is collected and stored in the Coiote?`

Reviewer: The filename should be changed too, I think.

> The Coiote IoT Device Management Platform enables you to gather diverse data types through the standardized data model offered by the LwM2M protocol.

Reviewer: `gather data through data model`? Maybe it's correct, I don't know, but from my point of view, `gather data inside data model` or something.

Within the platform, you'll discover, for example:

- Device Information
- Connectivity Status
- Firmware and Software Versions
- Configuration Data
- Sensor Data
- Location Data
- Security Information
- Battery Status

> It equips you with everything needed to monitor device health and collect essential data for IoT applications or external platforms.

Reviewer: ```suggestion
It equips you with everything needed to monitor device's health and collect essential data for IoT applications or external platforms.
```
But I'm not sure.
